cypress/integration/version/version_route.ts

```diff
@@ -86,4 +87,26 @@ describe("Version route", () => {
       cy.location("search").should("include", "statuses=will-run");
     });
   });
+
+  describe("Page title", () => {
+    before(() => {
+      cy.preserveCookies();
+      cy.visit(versionRoute(versions[6]));
+    });
+    it("Should include a link to Jira", () => {
+      cy.dataCy("page-title")
+        .contains("a", "EVG-7425")
+        .should("have.attr", "href", "https:///browse/EVG-7425");
```

Comment:
nit. Maybe Should we update the backend to provide the jira base url so we can have a semi usable link in testing.
But not necessary for this ticket.